GUI/users.py

```diff
@@ -0,0 +1,58 @@
+import csv
+
+"""_summary_
+    
+    First digit of userID specifies user type
```

Comment:
This system works, is there a reason for using digits rather than creating a enum/string.